src/components/UserManagement/UserTableData.jsx

```diff
@@ -60,7 +61,7 @@ const UserTableData = React.memo((props) => {
           ? (props.user.reactivationDate.toLocaleString().split('T')[0]) : ''}
       </td>
       <td>
-        {props.user.endDate ? (props.user.endDate.toLocaleString().split('T')[0]) : 'N/A'}
+        {props.user.endDate ? (moment(props.user.endDate).format('YYYY-MM-DD')) : 'N/A'}

```

Comment:
This is what caused the date to show as a long number as shown in the link below
https://user-images.githubusercontent.com/18335122/132959248-5d9ef39a-7751-4a48-b56f-810307f93462.jpg